wdl/MELT.wdl

```diff
@@ -625,8 +628,9 @@ task RunMELT {
     echo "JVM memory: $JVM_MAX_MEM"
 
     # call MELT
+    cwd=$(pwd)
     "$MELT_SCRIPT" \
-      "~{sample_id}.bam" \
+      "$cwd/~{sample_id}.bam" \
```

Comment:
It's not clear to my why this is needed, can't you just do `"~{sample_id}.bam" \` ?

Reply:
This is corrected now.